docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -468,6 +468,24 @@ specified in the following form:
     {"key": "header2", "value": "value2"}
   ]
 
+Envoy supports adding static and dynamic values to the request headers. Supported dynamic values are:
+
+%CLIENT_IP%
+   The original client IP which is already added by envoy as a  
+   :ref:`X-Forwarded-For <config_http_conn_man_headers_x-forwarded-for>` request header. 
+
+%PROTOCOL%
+    The original protocol which is already added by envoy as a 
+    :ref:`X-Forwarded-Proto <config_http_conn_man_headers_x-forwarded-proto>` request header. 
+
```

Comment:
Are these the only two supported Kunal ? I remember you adding support for more variables. If that's the case, it might be instructive to change the example to use a different variable, and possibly mention that these are same as ones supported by access log